metadetect/lsst/util.py

```diff
@@ -817,39 +812,32 @@ def get_stats_mask(exp):
     # these will be ignored when finding the image standard deviation
     # stats_mask = ['BAD', 'SAT', 'EDGE', 'NO_DATA']
 
-    stats_mask = ['BAD', 'EDGE', 'DETECTED', 'DETECTED_NEGATIVE', 'NO_DATA']
-
-    if 'BRIGHT' in exp.mask.getMaskPlaneDict():
-        stats_mask += ['BRIGHT']
+    import lsst.afw.image as afw_image
+    afw_image.Mask.addMaskPlane('BRIGHT')
+    stats_mask = ['BAD', 'EDGE', 'DETECTED', 'DETECTED_NEGATIVE', 'NO_DATA', 'BRIGHT']
```

Comment:
There are changes in the logic here around BRIGHT handling. Are those inteneded?

Reply:
Yes, these are intentional changes but there is no logic change here. The change was required was to keep this function not depend on any specific `Exposure` object `exp` . Instead of looking if `exp` has a `BRIGHT` mask plane and adding to this list, we always add it to the list. If `exp` does not have it defined, then the `addMaskPlane` call above defines it but doesn't set the bit in any pixel and everything proceeds as before. If it has `BRIGHT` plane defined already, then `addMaskPlane` call doesn't do anything and everything proceeds as before as well.